src/nidcpower/metadata/functions_addon.py

```diff
@@ -42,6 +42,7 @@
     'GetExtCalLastDateAndTime':        { 'codegen_method': 'private',  },  # Public wrapper to allow datetime
     'GetSelfCalLastDateAndTime':       { 'codegen_method': 'private',  },  # Public wrapper to allow datetime
     'FetchMultiple':                   { 'codegen_method': 'private',  },  # Fancy Fetch Public wrapper
+    'MeasureMultiple':                 { 'codegen_method': 'private',  },  # Fancy Measure Public wrapper
```

Comment:
Minor: `FancyMeasureMultiple`
I think we'd want search / grep to find the mention so we should use the precise name. Also decreases ambiguity.

Reply:
Does this apply to all fancy wrappers in all drivers?

Reply:
I think that'd be best. Better done in own PR.

Reply:
#819